generate-release-vars.sh

```diff
@@ -84,6 +85,7 @@ source_ami_al2       = "$ami_name_al2_x86"
 source_ami_al2arm    = "$ami_name_al2_arm"
 source_ami_al2kernel5dot10    = "$ami_name_al2_kernel5dot10"
 source_ami_al2kernel5dot10arm = "$ami_name_al2_kernel5dot10arm"
+source_ami_al2_gpu   = "amzn2-ami-minimal-hvm-2.0.20230926.0-x86_64-ebs"
```

Comment:
Maybe we don't need to update generate-release-vars.sh since the default value for source_ami_al2_gpu is defined in variables.pkr.hcl?
```
variable "source_ami_al2_gpu" {
  type        = string
  description = "Amazon Linux 2 source AMI to build AL2GPU AMI from. This is a temporary override."
  default     = "amzn2-ami-minimal-hvm-2.0.20230926.0-x86_64-ebs"
}
```

Reply:
True. This is writing to the `release.autopkrvars.hcl` so that releaser can see the diff and write change log though, so I would not think it is a big deal to keep it.